go/chat/search_test.go

```diff
@@ -525,19 +524,11 @@ func TestChatSearchInbox(t *testing.T) {
 			}
 		}
 
-		verifyIndex := func(expectedIndex *chat1.ConversationIndex) {
```

Comment:
could we add a function to the indexer/store that rebuilds the old style map (or something similar) to keep this checking in? these tests covered the indexer storage in a addition to the search logic but now we have no verification of storage

Reply:
Is there any value in doing that check outside of making sure the searches do the right thing?

Reply:
just to sanity check and to make sure the structures are as we expect, I could see a search not returning a result even if some alias data remained present for example

Reply:
Ok I think I will push that to a different PR, I think this test more or less lets you know this works, and I think it will be non-trivial work to get this working like that again.

Reply:
yup that sounds good